apps/blockchain/lib/blockchain/configuration/homestead_test.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Blockchain.Configuration.HomesteadTest do
```

Comment:
Why does the Homestead configuration live here instead of the EVM? What do you think about moving it within the evm app?

Reply:
I'm totally ok with it. But some parameters will be used only in the blockchain app. For example, contract creation cost

Reply:
Yeah, it just seems strange to me that the different implementations may live in different apps. If some parameters are only used in blockchain and some are only used in the evm, does it make sense to have something else (I dont' know what) where we keep the fork configurations?